src/hooks/useVersion.tsx

```diff
@@ -0,0 +1,43 @@
+import type { ReactNode, FC } from 'react';
+import { createContext, useContext, useState } from 'react';
+
+export type VersionCtx = 'Beta-4' | 'Latest';
+
+const versionCtx = createContext<VersionCtx>('Beta-4');
+const setVersionCtx = createContext<(value: VersionCtx) => void>(() => {});
+
+export function useVersion() {
+  return useContext(versionCtx);
+}
+
+export function useSetVersion() {
+  return useContext(setVersionCtx);
+}
+
+interface VersionProviderProps {
+  children: ReactNode;
+}
+
+export const VersionProvider: FC<VersionProviderProps> = ({ children }) => {
+  const [state, setState] = useState<VersionCtx>(() => {
+    if (typeof window !== 'undefined') {
+      return (sessionStorage.getItem('version') as VersionCtx) || 'Beta-4';
+    }
+    return 'Beta-4';
```

Comment:
Would be good to make the version a env variable to make it simpler to update in the future

Reply:
[FRO-610](https://linear.app/fuel-network/issue/FRO-610/replace-all-instances-of-beta-4-with-env-variable)